parquet-pig/src/main/java/org/apache/parquet/pig/TupleWriteSupport.java

```diff
@@ -174,6 +180,8 @@ private void writeValue(Type type, FieldSchema pigType, Tuple t, int i) {
             bytes = ((DataByteArray)t.get(i)).get();
           } else if (pigType.type == DataType.CHARARRAY) {
             bytes = ((String)t.get(i)).getBytes("UTF-8");
+          } else if (pigType.type == DataType.DATETIME) {
+            bytes = convertDateTimeToInt96((DateTime) t.get(i));
```

Comment:
As we want to move away from INT96 to TIMESTAMP_MILLIS and TIMESTAMP_MICROS, shouldn't they be rather used here?